src/ESM.t.sol

```diff
@@ -115,22 +115,22 @@ contract ESMTest is DSTest {
     function test_full() public {
```

Comment:
Do we need this test anymore? Seems like you are testing that join does its math more than `full` at this point